### DISCUSSION TITLE
Could limit user file transfer?

### DISCUSSION THREAD

**Discussion by akong77** [rustdesk/rustdesk-server#340] [General]
Hello,
The user can enable or disable to file transfer.Could I limit by server?Thanks a lot.

**Comment by rustdesk** [comment]
But in Pro only, https://rustdesk.com/docs/en/self-host/rustdesk-server-pro/strategy/
![image](https://github.com/rustdesk/rustdesk-server/assets/71636191/0b2b75c5-1a60-4c25-b16c-88b09ff2a797)

**Reply by akong77** [reply]
OK,
I has see Basic self-hosting plan.Is this have web console?
